<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleFilterWidgetGui</class>
 <widget class="QWidget" name="ParticleFilterWidgetGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
   <item>
    <widget class="QWidget" name="playerWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
      <item>
       <widget class="QLabel" name="display_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">Background-color: #000;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="buttonsLeft_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="browse_pushButton">
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_pushButton">
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pause_pushButton">
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveTarget_pushButton">
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Save Target</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="buttonsRight_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>PF Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="12" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Top N particles</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSlider" name="particlesToDisplaySlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="particlesToDisplayLabel">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="numTrackersSpinBox">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>256</number>
           </property>
           <property name="singleStep">
            <number>4</number>
           </property>
           <property name="value">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="particleHeightLbel">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Particle Height</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="particleCountSpinBox">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="24" column="0">
          <widget class="QPushButton" name="setupPF_pushButton">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Setup Particle Filter</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Futura</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ROI Confidence:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="17" column="0" rowspan="3" colspan="2">
          <widget class="QFrame" name="predictorFrame">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLabel" name="roi_class">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="stepSizeSizeLabel">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Step size (conf map)</string>
           </property>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QSpinBox" name="stepSizeSizeSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
           <property name="singleStep">
            <number>4</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="21" column="0" colspan="2">
          <widget class="QPushButton" name="computeConfMapPushButton">
           <property name="text">
            <string>compute conf map</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="HistogramSizelabel_3">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>DBSCAN MIN PTS</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="HistogramSizelabel_2">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>DBSCAN EPS</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Futura</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ROI Class:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="iterCountLabel">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string># Iterations</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="particleWidthLbel">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Particle Width</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="particleCountLabel">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string># Particles</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="particleWidthLSpinBox">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>55</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="iterCountSpinBox">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="numTrackersLabel">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string># Trackers/obj</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="roi_confidence">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="particleHeightLSpinBox">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>55</number>
           </property>
          </widget>
         </item>
         <item row="23" column="0">
          <widget class="QListWidget" name="targets_ListWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>10000</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Helvetica Neue</family>
            </font>
           </property>
          </widget>
         </item>
         <item row="22" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Targets :</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSpinBox" name="dbscanMinPtsSizespinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>256</number>
           </property>
           <property name="singleStep">
            <number>4</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="dbscanEpsDoubleSpinBox">
           <property name="value">
            <double>8.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browse_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onActionOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onActionPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pause_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onActionPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onHorizontalSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onHorizontalSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>particleCountSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onParticleCountChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterCountSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onIterationCountChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>particleWidthLSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onParticleWidthChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>particlesToDisplaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onSetParticleCountSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1064</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveTarget_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onActionSaveTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numTrackersSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onHistSizeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setupPF_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onActionSetupPF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1062</x>
     <y>704</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onHorizontalSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>particleHeightLSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onParticleHeightChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>1101</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stepSizeSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onStepSizeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>1101</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>computeConfMapPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onActionDispConfMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1069</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>1101</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbscanEpsDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onDBSCANEpsChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbscanMinPtsSizespinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ParticleFilterWidgetGui</receiver>
   <slot>onDBSCANMinPtsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionOpen()</slot>
  <slot>onActionPlay()</slot>
  <slot>onActionPause()</slot>
  <slot>onHorizontalSliderPressed()</slot>
  <slot>onHorizontalSliderReleased()</slot>
  <slot>onHorizontalSliderMoved(int)</slot>
  <slot>onParticleCountChange(int)</slot>
  <slot>onIterationCountChange(int)</slot>
  <slot>onParticleWidthChange(int)</slot>
  <slot>onSetParticleCountSliderMoved(int)</slot>
  <slot>onActionSaveTarget()</slot>
  <slot>onHistSizeChanged(int)</slot>
  <slot>onActionSetupPF()</slot>
  <slot>onParticleHeightChange(int)</slot>
  <slot>onStepSizeChanged(int)</slot>
  <slot>onActionDispConfMap()</slot>
  <slot>onDBSCANEpsChanged(double)</slot>
  <slot>onDBSCANMinPtsChanged(int)</slot>
 </slots>
</ui>
